scripts/services.js

```diff
@@ -2206,17 +2206,15 @@ var trackerCaptureServices = angular.module('trackerCaptureServices', ['ngResour
                     entity.inactive = row[6] !== "" ? row[6] : false;
                     entity.followUp = isFollowUp;
 
-                    if(grid.headers[grid.headers.length-2].name == 'lastdate'){
-                        entity.lastdate = row[row.length-2];
-                    }
-
                     if(grid.headers[grid.headers.length-1].column == 'TransferStatus'){
                         //entity.followUp =  entity.followUp || row[row.length-1] == "ACTIVE";
                     }
                     
                     for (var i = 7; i < row.length; i++) {
                         if (row[i] && row[i] !== '') {
                             var val = row[i];
+                            if(!grid.headers[i]) 
+                            {debugger};

```

Comment:
debugger?

Reply:
Nice catch. This was a bit of debug code.